panther_bringup/launch/bringup.launch.py

```diff
@@ -255,7 +255,7 @@ def generate_launch_description():
     )
 
     other_action_timer = TimerAction(
-        period=20.0,
+        period=7.0,
```

Comment:
Can u motivate the new changes? Are they tested on hardware?

Reply:
I change it because it taking, so long to make quick test. I didn't want to commit this thanks.